src/javascript/app_2/Stores/Modules/Trading/Helpers/__tests__/barrier.js

```diff
@@ -0,0 +1,131 @@
+import { expect }              from 'chai';
+import React                   from 'react';
+import { buildBarriersConfig } from '../barrier';
+
+describe('buildBarriersConfig', () => {
+    it('Returns Undefined if contract has no barriers', () => {
+        const contract = {
+            "barrier_category":"euro_atm",
+            "contract_category":"callput",
+            "contract_category_display":"Up\/Down",
+            "contract_display":"Higher",
+            "contract_type":"CALL",
+            "exchange_name":"FOREX",
+            "expiry_type":"daily",
+            "market":"forex",
+            "max_contract_duration":"365d",
+            "min_contract_duration":"1d",
+            "sentiment":"up",
+            "start_type":"spot",
+            "submarket":"major_pairs",
+            "underlying_symbol":"frxAUDJPY"
```

Comment:
would it be possible to define a contract object (without barriers) before running the tests, and only extends barriers when it is needed?
e.g: 
```
// global object
const contract_obj = {
    barrier_category: "euro_atm",
    contract_category: "callput",
    ...etc
};

it('...', () => {
    const contract = { 
           ...contract_obj, 
           barriers: 0, 
    }
});
```